config/qualifications.yml

```diff
@@ -8,8 +8,8 @@ description: >
   Since you will be trusted with university-owned computers and potentially
   sensitive data, you must review these documents
 subtasks:
-  - '[Security Checklist](https://www.umass.edu/it/support/security/security-checklist-university-owned-computers)'
-  - '[Data Categorization](https://www.umass.edu/it/security/data-categorization)'
+  - '[Security and Safety](https://www.umass.edu/afsystems/basic-page/guides-security-and-safety/330)'
```

Comment:
This page contains the following things:
* A link to the "Service Categorizations" page linked to below
* A broken link
* A ton of information on Docusign - unneeded
* A barely-understandably PDF copy of some notes from a presentation that Travis gave in 2012
* A how-to on how to use MS Office document encryption - unneeded

Sounds like we only have one "document" to review at the moment?